docs/design-time-builds.md

```diff
@@ -81,21 +82,21 @@ You can force Visual Studio to show the results of a design-time build using the
 #### Visual Studio 2015 or below
 
 1. Delete the `.vs` directory that sits alongside the solution that is experiencing the problem
-2. Start a Developer Command Prompt for VS2015
+2. Start a _Developer Command Prompt for VS2015_
 3. At the prompt, run `SET TRACEDESIGNTIME=true`
-4. At the prompt, run 'devenv'
+4. At the prompt, run `devenv`
 5. Open the solution
-6. Under %TEMP%, look for [RANDOMGUID].designtime.log files, these will contain the results of the design-time build. If running Visual Studio 2015 Update 2 or higher, the name of the project and design-time target that is being called will also be included in the file name.
+6. Under `%TEMP%`, look for `[RANDOMGUID].designtime.log` files, these will contain the results of the design-time build. If running Visual Studio 2015 Update 2 or higher, the name of the project and design-time target that is being called will also be included in the file name.
 
 #### Visual Studio 2017 or later
 
 1. Install the [Project System Tools](https://marketplace.visualstudio.com/items?itemName=VisualStudioProductTeam.ProjectSystemTools) extension
 2. In Visual Studio, choose the `View > Other Windows > Build Logging` menu item.
 3. Click on the "play" button.
 
-This will cause design time builds to show up in the build logging tool window. If you have the [MSBuild Binary and Structured Log Viewer](http://msbuildlog.com/) installed, you can double-click on a log to view it in the viewer, otherwise you can right-click and choose `Save As...` to save the log in the new [binary log format](https://github.com/Microsoft/msbuild/wiki/Binary-Log).
+This will cause design-time builds to show up in the build logging tool window. If you have the [MSBuild Binary and Structured Log Viewer](http://msbuildlog.com/) installed, you can double-click on a log to view it in the viewer, otherwise you can right-click and choose `Save As...` to save the log in the new [binary log format](https://github.com/Microsoft/msbuild/wiki/Binary-Log).
 
-### Diagnosing why a design-time build is failing or taking too long
+### Diagnosing failing or slow design-time builds
```

Comment:
I think there's an index at start of file that needs to be updated.

Reply:
The index links to the H2, not the H3.